.github/workflows/ci.yml

```diff
@@ -5,131 +5,68 @@ on:
   pull_request:
 
 env:
-  COMPOSER_ALLOW_SUPERUSER: '1' # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
-  COMPOSER_INSTALLER_COMMIT: fb22b78362d31c0d2bf516d1f8cdfd2745caa431
-  COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN  }}
+  COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   COVERAGE: '0'
-  EXT_MONGODB_VERSION: '1.7.5'
-  EXT_PCOV_VERSION: '1.0.6'
 
 jobs:
   php-cs-fixer:
-    name: PHP-CS-Fixer (PHP ${{ matrix.php }})
     runs-on: ubuntu-latest
-    container:
-      image: php:${{ matrix.php }}-alpine
-      options: >-
-        --tmpfs /tmp:exec
-        --tmpfs /var/tmp:exec
-    strategy:
-      matrix:
-        php:
-          - '7.4'
-      fail-fast: false
-    timeout-minutes: 5
     env:
       PHP_CS_FIXER_FUTURE_MODE: '1'
-      PHP_CS_FIXER_VERSION: '^2.14'
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
-      - name: Install system packages
-        run: |
-          apk add \
-            unzip \
-            git openssh
-      - name: Disable PHP memory limit
-        run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
-      - name: Install Composer
-        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/$COMPOSER_INSTALLER_COMMIT/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
-      - name: Cache Composer packages
-        uses: actions/cache@v2
+        uses: actions/checkout@v2
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
         with:
-          path: ~/.composer/cache
-          key: composer-php${{ matrix.php }}-php-cs-fixer-${{ github.sha }}
-          restore-keys: |
-            composer-php${{ matrix.php }}-php-cs-fixer-
-            composer-php${{ matrix.php }}-
-            composer-
-        continue-on-error: true
-      - name: Install Symfony Flex
-        run: |
-          composer global require --prefer-dist --no-progress --no-suggest --ansi \
-            symfony/flex
-      - name: Install PHP-CS-Fixer
-        run: |
-          composer global require --prefer-dist --no-progress --no-suggest --ansi \
-            friendsofphp/php-cs-fixer:"${PHP_CS_FIXER_VERSION}"
-      - name: Cache PHP-CS-Fixer results
+          php-version: '7.4'
+          extensions: intl, openssl, mbstring 
+          ini-values: memory_limit=-1
+          tools: pecl, composer, flex, php-cs-fixer
+          coverage: none
+      - name: Get composer cache directory
```

Comment:
Do we really need the composer cache for this job?

Reply:
right we could even skip the composer installation